relay/abi.go

```diff
@@ -171,12 +172,32 @@ func generatePostedKeyMsg(q *uploader.Queue, event PostedKey) (err error) {
 	return
 }
 
+func generateDeletedFileTreeMsg(q *uploader.Queue, event DeletedFileTree) (err error) {
+	log.Printf("Event details: %+v", event)
+	evmAddress = event.From.String()
+
+	h := sha256.New()
+	h.Write([]byte(event.Account))
+	accountHash := h.Sum(nil)
+
+	relayedMsg = evmTypes.ExecuteMsg{
+		DeleteFileTree: &evmTypes.ExecuteMsgDeleteFileTree{
+			HashPath: MerklePath(event.HashPath),
```

Comment:
We don't need to do the MerklePath stuff on the relayer, we can just do this on the client

Reply:
Ah okay, I originally thought we should do it in mulberry since postFile was calculating merkles